fcrepo-clients/src/main/java/edu/unc/lib/dl/util/TripleStoreQueryServiceFusekiImpl.java

```diff
@@ -0,0 +1,363 @@
+import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
+import org.apache.jena.query.Query; 
+import org.apache.jena.query.QueryExecution; 
+import org.apache.jena.query.QueryExecutionFactory; 
+import org.apache.jena.query.QueryFactory; 
+import org.apache.jena.query.QuerySolution; 
+import org.apache.jena.query.ResultSet;
+import org.apache.jena.rdf.model.Resource;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import edu.unc.lib.dl.fcrepo4.PIDs;
+import edu.unc.lib.dl.fedora.PID;
+import edu.unc.lib.dl.rdf.PcdmModels;
+import edu.unc.lib.dl.util.ContentModelHelper.Model;
+
+public class TripleStoreQueryServiceFusekiImpl implements
+		TripleStoreQueryService {
+	private static final Logger log = LoggerFactory.getLogger(TripleStoreQueryServiceFusekiImpl.class);
+	private static final String tripleStoreEndpoint = "http://localhost:8080/fuseki/test/query";
+	
+	private String fusekiEndpointURL;
+	private CloseableHttpClient httpClient;
+	private ObjectMapper mapper;
+	private PID collections;
+	
+	private final HttpClientConnectionManager multiThreadedHttpConnectionManager;
+			
+	public TripleStoreQueryServiceFusekiImpl() {
+		this.multiThreadedHttpConnectionManager = new PoolingHttpClientConnectionManager();
+		this.httpClient = HttpClients.custom()
+				.setConnectionManager(multiThreadedHttpConnectionManager)
+				.build();
+		this.mapper = new ObjectMapper();
+		this.collections = null;
+	}
+
+	public void destroy() {
+		this.httpClient = null;
+		this.mapper = null;
+		this.multiThreadedHttpConnectionManager.shutdown();
+	}
+
+	@Override
+	public List<PID> fetchAllContents(PID key) {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	@Override
+	public List<PID> fetchChildren(PID key) {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	@Override
+	public Map<String, PID> fetchChildSlugs(PID parent) {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	@Override
+	public List<PathInfo> fetchChildPathInfo(PID parent) {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	@Override
+	public List<PID> fetchChildContainers(PID key) {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	@Override
+	public List<PID> fetchByPredicateAndLiteral(String predicateURI,
+			String literal) {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	@Override
+	public List<PID> fetchByPredicateAndLiteral(String predicateURI,
+			PID pidLiteral) {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	@Override
+	public PID fetchByRepositoryPath(String path) {
+		// TODO Auto-generated method stub
+		return null;
+	}
+	
+	@Override
+	public PID fetchContainer(PID child, Resource membershipRelation) {
+		Resource relationship = (membershipRelation != null) ? membershipRelation : PcdmModels.hasMember;
+		
+		String queryString = String.format(
+				"select ?pid where { ?pid <%1$s> <%2$s> }",
```

Comment:
Generally, `<>` is a relative reference to the current object, <.> to the parent.  In this case its just used to wrap non-literals so that sparql can tell that the predicate and object are not strings, but are instead a property and a resource respectively.  The ? is part of the sparql query syntax, it just denotes a query variable